en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+# Derive Insights
+
+This section explains how you can derive insights about the performance of your Choreo instance via the Analytics Portal.
+
+## Access the Analytics Portal
+
+To access the Analytics Portal, into the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/), and click **Try now** on the **Insights** card.
```

Comment:
This is not true anymore. The insights charts appear as soon as you click on the insights menu on the left navigation bar.

Reply:
Changed this. Please check.